src/Symfony/Component/Form/Extension/Core/DataTransformer/DateTimeToArrayTransformer.php

```diff
@@ -21,16 +21,16 @@
  */
 class DateTimeToArrayTransformer extends BaseDateTimeTransformer
 {
-    private $pad;
+    private bool $pad;
 
-    private $fields;
-    private $referenceDate;
+    private array $fields;
+    private \DateTimeInterface $referenceDate;
 
     /**
-     * @param string $inputTimezone  The input timezone
-     * @param string $outputTimezone The output timezone
-     * @param array  $fields         The date fields
-     * @param bool   $pad            Whether to use padding
+     * @param string|null $inputTimezone The input timezone
+     * @param string|null $outputTimezone The output timezone
+     * @param array|null  $fields The date fields
+     * @param bool        $pad Whether to use padding
```

Comment:
alignment should be fixed
but before: what about just dropping the phpdoc? or removing the redundant lines?